unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -1987,6 +1987,16 @@ abstract public class BaseFPSAgentController {
             actionFinished(true);
         }
 
+        protected IEnumerator waitForSecondsRealtime(int seconds) {
```

Comment:
Don't see this being referenced anywhere

Reply:
It's being used by the new `Sleep` action which is used by the new tests that check that make sure the timeout code works properly.